src/google/protobuf/compiler/csharp/csharp_helpers.h

```diff
@@ -76,8 +76,6 @@ std::string GetUmbrellaClassUnqualifiedName(const FileDescriptor* descriptor);
 // not including the GetFileNamespace part).
 std::string GetUmbrellaClassNestedNamespace(const FileDescriptor* descriptor);
 
-std::string GetClassName(const Descriptor* descriptor);
```

Comment:
This header was defined twice - once in csharp_names.h and once here in csharp_helper.h.

Obviously when we were exposing GetClassName publicly in csharp_names.h, we forgot to remove it from here.